1-html-css/flexbox/style.css

```diff
@@ -0,0 +1,85 @@
+
+
+.challenge-1 {
+    display: flex;
+    flex-direction: row;
+}
+
+.challenge-2 {
+    display: flex;
+    flex-direction: row-reverse;
+
+}
+
+.challenge-3 {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    
+}
+
+.challenge-4 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+.challenge-5 {
+    display: flex;
+    justify-content: center;
+    gap: 20px;
+}
+
+/* 
+Tip: This is called the "child selector". 
+It selects all .bubble elements that are direct
+children of .challenge-6 elements. 
+*/
+.challenge-6 .bubble {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.challenge-7 {
+    display: flex;
+    flex-direction: row;
+}
+
+
+.challenge-7 .red {
+    flex-grow: 1;
+}
+
+.challenge-8 {
+display: flex;
+flex-direction: column;
+align-items: center;
+}
+
+.challenge-8 .red {
+margin-left: auto;
```

Comment:
I wanted you to do this with flexbox properties! I also meant "the end" as in the bottom, which in this case is also flex-end. I should change it to say "bottom" to be more clear.